validate/validate.go

```diff
@@ -424,7 +424,11 @@ func (v *Validator) CheckPlatform() (msgs []string) {
 		}
 	}
 
-	if v.platform != "windows" {
+	if v.platform == "windows" {
+		if v.spec.Windows == nil {
+			msgs = append(msgs, "'windows' MUST be set when platform is `windows`")
+		}
+	} else {
 		if v.spec.Windows != nil {
 			msgs = append(msgs, fmt.Sprintf("'windows' MUST NOT be set when target platform is %q", v.platform))
```

Comment:
This is [no longer true in rc6][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc6/config.md#L348-L349